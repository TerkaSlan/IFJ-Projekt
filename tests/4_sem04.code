/*This test should throw error because of more arguments in calling function than in declaration */

class Main
{
	static void run()
	{
		int a = someFunc(10,15);
	} // end of static void run()
	
	static int someFunc(int x)
	{
		return (x+42);
	}
} // end of class Main