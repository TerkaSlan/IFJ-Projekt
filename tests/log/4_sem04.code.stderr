ERROR: Semantic - type - Line: 7 - Too much paramaters in function call
